Data/Versions.hs

```diff
@@ -490,31 +494,21 @@ instance Semantic PVP where
 --
 -- Examples of @Version@ that are not @SemVer@: 0.25-2, 8.u51-1, 20150826-1, 1:2.3.4
 data Version = Version
-  { _vEpoch  :: Maybe Word
-  , _vChunks :: [VChunk]
-  , _vRel    :: [VChunk] }
+  { _vEpoch  :: !(Maybe Word)
+  , _vChunks :: !(NonEmpty VChunk)
```

Comment:
A `Version` without any chunks is meaningless, and no longer allowed. Unfortunately this kills the `Monoid` instance for this type, and so the instance has been removed.